test/unit/cmap/handshake/client_metadata.test.ts

```diff
@@ -70,7 +73,7 @@ describe('client metadata module', () => {
       context('unrelated environments', () => {
         before(() => {
           // aws
-          process.env.AWS_EXECUTION_ENV = 'non-empty-string';
+          process.env.AWS_EXECUTION_ENV = 'AWS_Lambda_non-empty-string';
```

Comment:
can we use the same spelling in all the other versions of this value (`AWS_Lambda_non empty string -> AWS_Lambda_non-empty-string`)? it reads funny when you have a mix of underscores and spaces

Reply:
fixed!